pkg/build/node/node.go

```diff
@@ -160,7 +160,7 @@ func (c *BuildContext) populateBits(buildDir string) error {
 }
 
 // BuildContainerLabelKey is applied to each build container
-const BuildContainerLabelKey = "io.k8s.test-infra.kind-build"
+const BuildContainerLabelKey = "io.k8s.sigs.kind.build"
```

Comment:
Per: https://docs.docker.com/config/labels-custom-metadata/#key-format-recommendations
These should be reverse DNS for a domain we "own", `io.k8s.sigs` seems natural but really we are `kind/ios.sigs.k8s` which is not a domain, so -> `io.k8s.sigs.kind.` is the new common prefix / namespace for our docker object labels

Reply:
@munnerz this is a breaking change of sorts FYI, but we don't exactly support long-lived clusters to begin with and this detail was not documented as a public API (yet?)